src/hotspot/share/compiler/oopMap.cpp

```diff
@@ -564,7 +564,7 @@ void OopMapSet::trace_codeblob_maps(const frame *fr, const RegisterMap *reg_map)
   tty->cr();
   fr->print_on(tty);
   tty->print("     ");
-  cb->print_value_on(tty);  tty->cr();
```

Comment:
We could minimize the number of files changed if we keep print_value_on() for compatibility:
```
void print_value_on(outputStream* st) const { print_value_on_v(st); }

```